### PR TITLE
Default Settings for HDRP

### PR DESCRIPTION
**Pull request by fredpointzero**
### Purpose of this PR
Default settings are now stored in HDRenderPipelineAsset and displayed in a dedicated panel.
Only the one assigned in `GraphicsSettings.renderPipelineAsset` will apply its default settings.

TODO: Use at runtime the default settings from the correct HDRP Asset

---
### Release Notes
- Added Default Settings Panel
- Use default settings in the HDRP Asset assigned in `GraphicsSettings.renderPipelineAsset`
- Shader Stripper use only HDRP Assets for the current platform target (including relevant quality levels)

---
### Testing status
**Katana Tests**: 

**Manual Tests**: 
- [ ] Opened test project + Run graphic tests locally
- [ ] Built a player
- [ ] Checked new UI names with UX convention
- [ ] Tested UI multi-edition + Undo/Redo
- [ ] C# and shader warnings (supress shader cache to see them)
- [ ] Checked new resources path for the reloader (in devloper mode, you have a button at end of resources that check the pathes)
- Other: 

**Automated Tests**: 

---
### Overall Product Risks
**Technical Risk**: Low

**Halo Effect**: Medium

---

### CODE REVIEW

File: com.unity.render-pipelines.high-definition/Editor/BuildProcessors/HDRPPreprocessShaders.cs [hunk 1]
```diff
@@ -274,7 +315,16 @@ static void GetAllValidHDRPAssets()
                 Resources.FindObjectsOfTypeAll<HDRenderPipelineAsset>()
                 .Where( a => !_hdrpAssets.Contains(a) )
                 );
-            
+
+            // Discard duplicate entries
+            using (HashSetPool<HDRenderPipelineAsset>.Get(out var uniques))
+            {
```
Comment (sebastienlagarde) [inline com.unity.render-pipelines.high-definition/Editor/BuildProcessors/HDRPPreprocessShaders.cs]: question: is is done at each call for each shader or only one time for a build ?

File: com.unity.render-pipelines.high-definition/Editor/Material/GeneralNodes.meta [hunk 2]
```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 53a596a9943af0b45800244f08688b43
+guid: f1a7f30e2a21b3f41b550005682865bd
```
Comment (sebastienlagarde) [inline com.unity.render-pipelines.high-definition/Editor/Material/GeneralNodes.meta]: ?

File: com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Settings/DefaultSettings.cs [hunk 3]
```diff
@@ -20,9 +20,11 @@ static DefaultSettings()
 
         private static Volume s_DefaultVolume = null;
 
-        public const string defaultVolumeProfileFileStem = "DefaultSettingsVolumeProfile";
+        public static HDRenderPipelineAsset hdrpAssetWithDefaultSettings
+            => GraphicsSettings.renderPipelineAsset is HDRenderPipelineAsset hdrpAsset ? hdrpAsset : null;
```
Comment (sebastienlagarde) [inline com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Settings/DefaultSettings.cs]: isn't this always true?

GraphicsSettings.renderPipelineAsset is HDRenderPipelineAsset

§I suppose that if you switch to LW we never call the code here?

File: com.unity.render-pipelines.high-definition/Editor/BuildProcessors/HDRPPreprocessShaders.cs (hunk 1 above)
Comment (fredpointzero) [reply com.unity.render-pipelines.high-definition/Editor/BuildProcessors/HDRPPreprocessShaders.cs]: Its only one time per build

File: com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Settings/DefaultSettings.cs (hunk 3 above)
Comment (fredpointzero) [reply com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Settings/DefaultSettings.cs]: It should be always true, if you switch to LW, the code should never be called.

**Comment by sebastienlagarde** [comment]
katana link: https://katana.bf.unity3d.com/projects/com.unity.render-pipelines/builders?ScriptableRenderLoop_branch=HDRP%2FDefaultFrameSettings&automation-tools_branch=master&unity_branch=trunk#

**Comment by sebastienlagarde** [comment]
katana is green , merging

**Pull request merged**